flutter/lib/main.dart

```diff
@@ -32,8 +33,10 @@ Future<void> main() async {
 
 Future<void> launchUi() async {
   await initDeviceInfo();
+  await FirebaseManager.staticInit();
+  final firebaseManager = await FirebaseManager.create();
```

Comment:
Must `FirebaseManager.staticInit()` always be called before `FirebaseManager.create()`? Can we integrate `FirebaseManager.staticInit()` into `FirebaseManager.create()`?

Reply:
`FirebaseManager.staticInit` must be called exactly once. It will throw an error if called for a second time.
In theory it's possible to create several instances, so I don't think we should integrate `staticInit` into `create` method. It's just a coincidence that we call both functions in one place.
That being said, `create` in its current implementation will also throw an error if called several times, so this must be fixed. I'll make it a singleton to reflect how underlying library works.